modules/acm_promotion/src/Plugin/Block/AcmPromotionBlock.php

```diff
@@ -159,19 +204,20 @@ public function build() {
    *
    * This method loads the active promotion nodes.
    *
-   * @return Drupal\node\Entity\Node[]
+   * @return \Drupal\node\Entity\Node[]
```

Comment:
Should this be `NodeInterface[]` ?

Reply:
Yes! as per standard, type should be interface. I only fixed the errors thrown by deprecation check. I will update to use entity interface

Reply:
PR updated! Thanks!